### ISSUE TITLE
Splenic lesion

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6851]
**Preferred term label:**
Splenic lesion

**Synonyms**
Lesions of the spleen

**Definition (free text, please give PubMed ID)**
A lesion originating within the spleen.; PMID: 28144107

**Parent term (use hpo.jax.org/app)**
Abnormal spleen morphology

**Comment by pnrobinson** [comment]
@johannaest the word "lesion" is so non-specific that it is a synonym of "Abnormal spleen morphology" or very nearly so. I am going to add this as a related synonym. If you have more information about the specific lesion please reopen this issue or make a new issue.
